hack/csv-generate.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+export GOROOT=$(go env GOROOT)
+TMP_CSV_VERSION="9.9.9"
```

Comment:
why this and not, say, `0.0.0`? maybe you fixed already on later patches

Reply:
it's a temporary csv that gets deleted. i can try 0.0.0 and see if that works too